packages/next/next-server/server/next-server.ts

```diff
@@ -1177,18 +1115,9 @@ export default class Server {
         query.__nextFallback = 'true'
```

Comment:
It's not super clear to me how this isn't tainting the `doRender` call in `canary` already.